src/authorization/TKBrowser.h

```diff
@@ -0,0 +1,54 @@
+//
+//  TKBrowser.h
+//  TokenSdk
+//
+//  Created by Sibin Lu on 12/12/17.
+//  Copyright © 2017 Token Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class TKBrowser;
+
+/**
+ * TKAuthorizationEngine observes TKBrowser with this protocol
+ */
+@protocol TKBrowserDelegate
+@required
+/**
+ * Return Yes if the browser shall load this request; Otherwise the browser will discard the request.
```

Comment:
I don't understand what this method accomplishes. When would you want to discard the request?